site/app/templates/grading/electronic/RubricPanel.twig

```diff
@@ -25,13 +25,14 @@
                     {# Verify all button #}
                     {% if show_verify_all %}
                         <div class="col-no-gutters">
+                            <label for="verify-all" tabIndex="0" style="display:none;">Verify All</label>
```

Comment:
This does not strike me as the right way to solve this. WAVE is flagging this input (and others) on needing a label to give assistance to screen readers on what the input does. By including `style="display: none"`, while the label exists and satisfies WAVE, it would still not appear for screen readers. Instead, we would want to "hide" the label by using a technique from [this page](https://webaim.org/techniques/css/invisiblecontent/), like [hiding it offscreen](https://webaim.org/techniques/css/invisiblecontent/#offscreen). This applies to all the other labels in this PR.